src/emmetHelper.ts

```diff
@@ -169,6 +171,10 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_');
```

Comment:
Also, can you update the tests in https://github.com/ramya-rao-a/vscode-emmet-helper/blob/master/src/emmetHelperTest.ts#L240 ?